src/ray/object_manager/plasma/store.cc

```diff
@@ -554,8 +556,15 @@ void PlasmaStore::PrintAndRecordDebugDump() const {
 }
 
 void PlasmaStore::RecordMetrics() const {
-  // TODO(sang): Add metrics.
+  absl::MutexLock lock(&mutex_);
+  RAY_LOG(DEBUG) << "Exporting plasma store metrics.";
   object_lifecycle_mgr_.RecordMetrics();
+
+  // Schedule the next metric export callback
+  metrics_timer_ = execute_after(
```

Comment:
We should setup this once, not each iteration of RecordMetrics, right?

Reply:
I believe this one works kind of recursively - I followed the pattern of the `stats_timer_` above mainly for consistency. We could also set up a periodic runner like you mention.

Reply:
Reverting.